### DISCUSSION TITLE
pcl2打不开

### DISCUSSION THREAD

**Discussion by YBB-Netizen** [Meloong-Git/PCL#4608] [提问] [answered]
pcl打不开下载完成打开显示这个界面
![微信截图_20240827163820](https://github.com/user-attachments/assets/ddf2b4db-50ee-493f-8ba6-ffd40a175efd)

**Comment by heerxingen** [comment]
issue才是提bug的地方，不要在讨论区提bug诶（小声）

**Reply by Deep-Dark-Forest** [reply]
有的时候不确定是不是 bug 在 discussion 里问一下也是可以的

**Comment by Pigeon0v0** [comment]
疑似系统问题，目前正在远程看一眼...

**Comment by Pigeon0v0** [comment, marked as answer]
忘记更新了，这位在系统镜像就地升级后解决
